SRTNet.h

```diff
@@ -174,17 +174,18 @@ class SRTNet {
     SRTNet &operator=(SRTNet const &) = delete;  // Copy assign
     SRTNet &operator=(SRTNet &&) = delete;      // Move assign
 
-
-private:
-
-    //Internal variables and methods
-
     enum class Mode {
```

Comment:
Move this to the top of the class to keep public structures and enumerations before the methods.